src/main/java/backend/github/GitHubRepo.java

```diff
@@ -260,6 +262,23 @@ public boolean login(UserCredentials credentials) {
     }
 
     @Override
+    public Issue setMilestone(String repoId, int issueId, String issueTitle, Integer issueMilestone)
+            throws IOException {
+        Milestone gitHubMilestone = new Milestone();
+        if (issueMilestone != null) {
```

Comment:
We should catch this at a higher level, as early as possible. It's easier to understand if the null case is ruled out closer to where it came from.